pysatModels/models/ucar_tiegcm.py

```diff
@@ -0,0 +1,208 @@
+                      "Killeen, N. Maruyama, and J. Lyon, Initial results",
+                      "from the CISM coupled magnetosphere-ionosphere-",
+                      "thermosphere (CMIT) model: thermosphere ionosphere ",
+                      "responses, J. Atmos. Sol.-Terr. Phys., 66, 1425-1442,",
+                      "doi:10.1016/j.jastp.2004.04.008, 2004.")),
+            " ".join(("Solomon, S. C., and L. Y. Qian, Solar",
+                      "extreme-ultraviolet irradiance for general circulation",
+                      "models, J. Geophys. Res., 110, A10306,",
+                      "doi:10.1029/2005JA011160, 2005.")),
+            " ".join(("Qian, L., A. G. Burns, B. A. Emery, B. Foster, G. Lu,",
+                      "A. Maute, A. D. Richmond, R. G. Roble, S. C. Solomon,",
+                      "and W. Wangm, The NCAR TIE-GCM: A community model of",
+                      "the coupled thermosphere/ionosphere system, in",
+                      "Modeling the Ionosphere-Thermosphere System, AGU",
+                      "Geophysical Monograph Series, 2014."))]
+
+    self.references = "\n".join((refs))
+    logger.info(self.acknowledgements)
+    return
+
+
+# ----------------------------------------------------------------------------
+# Instrument functions
+#
+# Use local and default pysat methods
+
+# Set the list_files routine
+fname = 'tiegcm_icon_merg2.0_totTgcm.s_{day:03d}_{year:4d}.nc'
+supported_tags = {'': {'': fname}}
+list_files = functools.partial(pysat.instruments.methods.general.list_files,
+                               supported_tags=supported_tags)
+
+
+def load(fnames, tag=None, inst_id=None, **kwargs):
+    """Load TIE-GCM data using xarray.
+
+    Parameters
+    ----------
+    fnames : array-like
+        Iterable of filename strings, full path, to data files to be loaded.
+        This input is nominally provided by pysat itself.
+    tag : str or NoneType
+        Tag name used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default=None)
+    inst_id : str or NoneType
+        Instrument ID used to identify particular data set to be loaded.
+        This input is nominally provided by pysat itself. (default=None)
+    **kwargs : dict
+        Passthrough for additional keyword arguments specified when
+        instantiating an Instrument object. These additional keywords
+        are passed through to this routine by pysat.
+
+    Returns
+    -------
+    data : xarray.Dataset
+        pysat formatted xarray Dataset
+    meta : pysat.Metadata
+        Model run meta data
+
+    Note
+    ----
+    Any additional keyword arguments passed to pysat.Instrument
+    upon instantiation are passed along to this routine.
+
+    Examples
+    --------
+    ::
+
+        inst = pysat.Instrument('ucar', 'tiegcm')
+        inst.load(2019, 1)
+
+    """
+
+    data, meta = pysat.utils.load_netcdf4(fnames, pandas_format=False)
```

Comment:
```suggestion
    data, meta = pysat.utils.io.load_netcdf(fnames, pandas_format=False,
                                            epoch_name='time',
                                            decode_times=True)
```

This improves forward compatibility with pysat/pysat#983, but is not backwards-compatible with the current develop or main branches of pysat.  A version limit would be needed if 983 is adopted into the next pysat PR, or potentially some sort of try / except logic

Reply:
Alternatively, write this up as an issue and push 983 to the next pysat release.